scripts/qa_crowbarsetup.sh

```diff
@@ -4054,10 +4053,24 @@ function onadmin_crowbarrestore()
     if iscloudver 6plus ; then
         systemctl start crowbar.service
         wait_for 20 10 "onadmin_is_crowbar_api_available" "crowbar service to start"
-        # crowbarctl needs --anonymous to workaround a crowbarctl issue which leads to two api requsts
-        # per call (auth + actual request) which fails when running crowbarctl directly on the admin node
-        safely crowbarctl backup upload /tmp/$btarball --anonymous
-        safely crowbarctl backup restore $btarballname --anonymous --yes
+        case $restoremode in
+            with_upgrade)
+                # restore after upgrade has different workflow (missing APIs) than
+                #   a restore from a backup of the same cloud release
+                local http_code=`curl -X POST -F "file=@/tmp/$btarball" -s -o upgrade-start.txt -w "%{http_code}" $crowbar_api/installer/upgrade/start`
```

Comment:
I guess you can't write-out the `http_code` here as long as it is not defined yet